### PR TITLE
Adding Video Classifier wrapper

### PR DESCRIPTION
**Pull request by ushareng**

### CODE REVIEW

File: keras_nlp/src/models/video_classifier.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import keras
+
+from keras_nlp.src.api_export import keras_nlp_export
+from keras_nlp.src.models.task import Task
+
+
+@keras_nlp_export("keras_nlp.models.VideoClassifier")
+class VideoClassifier(Task):
```
Comment (mattdangerw) [inline keras_nlp/src/models/video_classifier.py:21]: Why do we need this? What models use this? What are the expected input formats?

Comment (ushareng) [reply keras_nlp/src/models/video_classifier.py:21]: This wrapper will be used by video classifier models like video_swin, input format will be `(depth, height, width, channel)`

**Comment by divyashreepathihalli** [comment]
@ushareng has the VideoSwin model been added yet? The video classifier should be added once we have the VideoSwin backbone in. That will help us verify the implementation.

**Comment by mattdangerw** [comment]
Going to close this. The commit history got crazy. We can still add a video classifier, but probably should add it along with the model that will use the task, rather than adding the task first.

**Pull request closed without merging**